src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,64 @@
  */
 
 public class Plane {
+    private final String brand;
+    private final String model;
+    private final int passengersCapacity;
+    private final int crewCount;
+    private final int takeOffWeightTon;
```

Comment:
what is the purpose of using `final` here?

Reply:
because after we create a plane this fields are constants. This is a specification of any plane. I think they shouldn't change